index.html

```diff
@@ -107,6 +107,7 @@ <h1 id="target">000</h1>
                 <button id="demo-btn">Demo</button>                                   
             </div>
         </div> 
+        <!-- Comment -->
```

Comment:
This is cool.  I didn't realize that browsers had added support for CommonJS module imports.  My only note here is that I think you could get away with a normal script import e.g. `<script src="js/index.js">` since you aren't importing anything in your JS.

Reply:
To be completely honest, I had that originally because it's how that Electron tutorial did it and at the time when I was just trying to get a grasp on it, I kept doing that way and didn't want to mess with it too much. But it does seem to work, even through the Electron window, using he normal way.